src/main/java/com/amazon/opendistroforelasticsearch/commons/ConfigConstants.java

```diff
@@ -35,5 +35,8 @@
     public static final String INJECTED_USER = "injected_user";
     public static final String OPENDISTRO_SECURITY_USE_INJECTED_USER_FOR_PLUGINS = "opendistro_security_use_injected_user_for_plugins";
     public static final String OPENDISTRO_SECURITY_SSL_HTTP_ENABLED = "opendistro_security.ssl.http.enabled";
-    public static final String OPENDISTRO_SECURITY_USER_AND_ROLES = "_opendistro_security_user_and_roles";
+    public static final String OPENDISTRO_SECURITY_USER_INFO_THREAD_CONTEXT = "_opendistro_security_user_info";
+
+    @Deprecated
+    public static final String OPENDISTRO_SECURITY_USER_AND_ROLES = OPENDISTRO_SECURITY_USER_INFO_THREAD_CONTEXT;
```

Comment:
You want to delete this string completely to force compile failure?

Reply:
Sure. I can do that.